python/pfs/drp/stella/extractSpectraTask.py

```diff
@@ -20,6 +22,12 @@ class ExtractSpectraConfig(pexConfig.Config):
                                doc="Mask pixels to ignore in extracting spectra")
     minFracMask = pexConfig.Field(dtype=float, default=0.0,
                                   doc="Minimum fractional contribution of pixel for mask to be accumulated")
+    doCrosstalk = pexConfig.Field(dtype=bool, default=False, doc="Correct for optical crosstalk?")
+    crosstalk = pexConfig.ListField(
+        dtype=float,
+        default=[4.41695363e-03, 1.26907573e-03, 6.37238677e-04, 3.99808286e-04],
+        doc="Optical crosstalk coefficients, in increasing distance from the fiber of interest",
```

Comment:
Maybe mention in the doc that the coefficients represent the fraction of a given fiber's flux that appears in the neighboring fibers.